src/soc/sifive/fu540/src/ux00.rs

```diff
@@ -196,7 +196,7 @@ pub fn ux00ddr_phy_fixup() -> u64 {
 
     //let mut fails: u64 = 0;
     let mut slicebase: u32 = 0;
-    let mut dq: u32 = 0;
+    //let mut dq: u32 = 0;
```

Comment:
Delete rather than comment out.

Reply:
I left it as commented because there is other code nearby (e.g. 2 lines above and inside for loop below) that is also commented (and some is even using this dq variable). I would like to suggest to leave it commented as is, and clean up all commented code in some followup PR. WDYT?

Reply:
SGTM